modules/shared/src/main/scala/io/constellationnetwork/schema/swap.scala

```diff
@@ -161,12 +161,17 @@ object swap {
   @derive(decoder, encoder, order, show)
   case class SpendTransaction(
     fee: SpendTransactionFee,
-    lastValidOrdinal: SnapshotOrdinal,
+    validUntilOrdinal: SnapshotOrdinal,
     allowSpendRef: Hash,
-    currency: Option[CurrencyId],
+    currencyId: Option[CurrencyId],
```

Comment:
IMO currency was a better name. We see it's an identifier by its type.

Reply:
I was following the names described in the notion docs
https://www.notion.so/Currency-Swapping-cf462ad9c01d4e93902429282e7267e8#e1668bee754d49bb9f8cfb2001fde0ea

Reply:
👍🏼

Reply:
Fine with updating notion doc, really either is fine here.